WordPress/twentytwentyone
# Remove borders in post navigation

Fixes #377, see https://github.com/WordPress/twentytwentyone/issues/429#issuecomment-708384060 — This removes the last border-bottom on `nav a` in favor of intentionally setting text-decoration. It also removes a few leftover `border-bottom: none` where (I believe?) they're not necessary anymore.

@carolinan Were there any other places where you've seen borders?

## Relevant technical choices:

I've moved the pagination & navigation styles out of the generic `nav a` selector and into their respective sass partials, so that it doesn't cause any unexpected side effects.

## Test instructions

This PR can be tested by following these steps:
1. View a few pages, archives, etc
2. Links should be styled as expected
3. Hover & focus on links, they should be styled correctly.

## Screenshots

![Screen Shot 2020-10-14 at 3 18 18 PM](https://user-images.githubusercontent.com/541093/96035974-cb2f6780-0e31-11eb-9a9b-f285d52bcdbf.png)
![Screen Shot 2020-10-14 at 3 14 20 PM](https://user-images.githubusercontent.com/541093/96035975-cbc7fe00-0e31-11eb-847e-40e540b48d7f.png)

This should be the only "big change", part of which is already underway in #439, this uses text-decoration now instead of borders.
![Screen Shot 2020-10-14 at 3 19 30 PM](https://user-images.githubusercontent.com/541093/96035971-ca96d100-0e31-11eb-9d7c-a7f0a9cc0bed.png)

<details>
<summary>Full page in Mac/Firefox</summary>

![Screen Shot 2020-10-14 at 15 13 23-fullpage](https://user-images.githubusercontent.com/541093/96035916-b6eb6a80-0e31-11eb-94f1-a9f001f3e47e.png)

</details>

## Quality assurance
